### PR TITLE
Sign in with Apple resource in v1.0

### PR DESCRIPTION
**Pull request by namkedia**
Add support to Sign in with Apple resource to the identityProviderBase in v1.0

### CODE REVIEW

File: api-reference/v1.0/resources/applemanagedidentityprovider.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+This type will inherit from [identityProviderBase](../resources/identityproviderbase.md).
+
+## Methods
+
+| Method       | Return Type  |Description|
+|:---------------|:--------|:----------|
+|[List](../api/identityproviderbase-list.md)|identityProviderBase collection|Retrieve all identity providers configured in a tenant including the Apple identity providers.|
+|[Create](../api/identityproviderbase-post-identityproviders.md)|appleManagedIdentityProvider |Create a new Apple identity provider configuration.|
+|[Get](../api/identityproviderbase-get.md) |appleManagedIdentityProvider |Retrieve properties of the Apple identity provider configuration.|
```
[the remark this answers is not in the document]
Comment (namkedia) [reply api-reference/v1.0/resources/applemanagedidentityprovider.md:25]: Ack

File: api-reference/v1.0/resources/applemanagedidentityprovider.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+## Properties
+
+|Property|Type|Description|
+|:---------------|:--------|:----------|
+|developerId|String|The Apple developer identifier. Required.|
+|serviceId|String|The Apple service identifier. Required.|
+|keyId|String|The Apple key identifier. Required.|
+|certificateData|String|The certificate data which is a long string of text from the certificate, can be null.|
+|id|String|The identifier of the identity provider. Inherited from [identityProviderBase](../resources/identityproviderbase.md). Read-only.|
+|displayName|String|The display name of the identity provider. Inherited from [identityProviderBase](../resources/identityproviderbase.md).|
```
Comment (FaithOmbongi) [inline api-reference/v1.0/resources/applemanagedidentityprovider.md:39]: Reordered the table alphabetically for easier scanning
```suggestion
|certificateData|String|The certificate data which is a long string of text from the certificate, can be null.|
|developerId|String|The Apple developer identifier. Required.|
|displayName|String|The display name of the identity provider. Inherited from [identityProviderBase](../resources/identityproviderbase.md).|
|id|String|The identifier of the identity provider. Inherited from [identityProviderBase](../resources/identityproviderbase.md). Read-only.|
|keyId|String|The Apple key identifier. Required.|
|serviceId|String|The Apple service identifier. Required.|
```

File: api-reference/v1.0/resources/applemanagedidentityprovider.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+---
+title: "appleManagedIdentityProvider resource type"
+description: "Represents apple identity provider in an Azure AD B2C tenant."
+ms.localizationpriority: high
```
Comment (FaithOmbongi) [inline api-reference/v1.0/resources/applemanagedidentityprovider.md:4]: Recommended loc priority for reference articles is medium
```suggestion
ms.localizationpriority: medium
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
@namkedia , we're missing the references to appleManagedIdentityProvider in the Methods table of identityProviderBase resource. For example, [in the beta topic](https://docs.microsoft.com/en-us/graph/api/resources/identityproviderbase?view=graph-rest-beta#methods), we mention that the return type for the Create, Get, Delete operations may also be an appleManagedIdentityProvider. I assume this is still so in the v1.0 topic?

**Pull request merged**